DATATYPES.md

```diff
@@ -33,62 +33,6 @@ The result set metadata currently just displays the Type ID for each column as a
 |                12 | Timestamp                        | Timestamp                               |
 |                13 | TimestampTz                      | TimestampTz                             |
 |                20 | Uuid                             | Uuid                                    |
```

Comment:
I didn't catch this before. With the current version of the protocol (3.5) UUID is returned as type char which has a Type ID of 8, compared to the latest version of the protocol which returns UUID with a type ID of 20.

Reply:
That essentially means that Uuid is not used because the server returns the oid for char instead.  I guess we should remove Uuid until we move to that later protocol version in order to prevent people from expecting the Uuid to be used.   Of course, the server will always return the value as Char, so regardless of whether we have it defined it won't impact usage.

Reply:
I agree we should remove it completely until then. Otherwise someone may think they can provide their own type parser based on this oid

Reply:
I've gone ahead and made a comment in the protocol bump issue I logged to update this and type parsers for uuid oid 20 once we pass 3.9